tests/data-files/catalogs/test-case-6/12cb17a8-ae08-469c-a2be-4d0619240014/9420e0fd-3731-40e8-93ff-5b6ad0067048/collection.json

```diff
@@ -76,7 +76,9 @@
       "label:assets": []
     }
   ],
-  "stac_extensions": [],
+  "stac_extensions": [
+    "https://stac-extensions.github.io/label/v1.0.0/schema.json"
+  ],
```

Comment:
This is curious - I'm not sure why there are `label:assets` in those properties. I don't think this collection should actually extend label, and I don't think `label:asssets` should be included in the links. Perhaps there's a bug in the migration script?

Reply:
It looks like that test case was added in response to #88, which was way back in PySTAC 0.3.3. It had the `label:assets` properties when it was created, so they must have been relevant at the time, but they definitely don't fit now.

I'll remove the Label extension references in that example.

Reply:
Fixed in f020601e502eec3feb309633c200df9c9f1bc8e0